databricks_cli/configure/config.py

```diff
@@ -37,11 +38,15 @@ def provide_api_client(function):
     """
     @six.wraps(function)
     def decorator(*args, **kwargs):
+        ctx = click.get_current_context()
+        command_name = "-".join(ctx.command_path.split(" ")[1:])
+        command_uuid = str(uuid.uuid1())
+        default_headers = {'cli-command-name': command_name, 'cli-command-uuid': command_uuid}
```

Comment:
Could you explain why this head is needed here? Do you know where in the logs we can extract such information?

Reply:
I'm not sure, I was going to ask you about that.

Reply:
I was thinking that a new metric would need to be defined